Sources/Kitura/RouterResponse.swift

```diff
@@ -121,6 +121,20 @@ public class RouterResponse {
         statusCode = .unknown
     }
 
+    deinit {
+        if !state.invokedEnd {
+            if !state.invokedSend && statusCode == .unknown {
```

Comment:
That is so if a handler has written to the response, but just didn't end the response, we send ok status, not service unavailable.